src/helpers/map.js

```diff
@@ -1,9 +1,31 @@
-import { tileTypes, PIXELS_PER_TILE } from '../constants';
+import { tileTypes, PPTL, CPTL } from '../constants';
 import { amBlue, amRed } from './player';
 import { assert, assertGridInBounds } from '../../src/utils/asserts';
 
 
 /*
+ * Initializes and returns a 2D array with the specified width, height, and
+ * default value.
+ *
+ * @param {number} width - the width of the initialized 2D array
+ * @param {number} height - the height of the initialized 2D array
+ * @param {number} defaultVal - the value to give each element in the initialized 2D array
+ */
+export function init2dArray(width, height, defaultVal = 0) {
```

Comment:
It got called before its declaration, which something yelled about. So I moved it.